Analyzer/test/condor/samples.py

```diff
@@ -24,8 +25,9 @@ def nSamples(self, name):
     def sampleList(self, name):
         files = self.lib.SC_samples(self.obj, name)
         names = self.lib.SC_samples_names(self.obj, name)
-        nEvts = self.lib.SC_samples_nEvts(self.obj, name)
-        list = [(files[i],names[i],nEvts[i]) for i in xrange(self.lib.SC_samples_size(self.obj, name))]
+        nGenEvts = self.lib.SC_samples_nGenEvts(self.obj, name)
```

Comment:
I guess we don't need to use the GenEvts in general?

Reply:
Ah, that is correct, here we do not need access to `nGenEvts` just put it for symmetry, but can remove.